addons/l10n_hr_edi/data/cron.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="ir_cron_mer_get_new_documents" model="ir.cron">
+        <field name="name">MojEracun: retrieve new documents</field>
+        <field name="interval_number">4</field>
+        <field name="interval_type">hours</field>
+        <field name="model_id" ref="model_account_edi_mojeracun_proxy_client_user"/>
+        <field name="code">model._cron_mer_get_new_documents()</field>
+        <field name="state">code</field>
+    </record>
+
+    <record id="ir_cron_mer_get_document_status_inbox" model="ir.cron">
+        <field name="name">MojEracun: check document status</field>
+        <field name="interval_number">4</field>
+        <field name="interval_type">hours</field>
+        <field name="model_id" ref="model_account_edi_mojeracun_proxy_client_user"/>
+        <field name="code">model._cron_mer_get_document_status_inbox()</field>
+        <field name="state">code</field>
+    </record>
```

Comment:
I'm not sure this cron (and the corresponding method) are needed unless the status on MER of a received document can change after it has been received.